sys/docker/alpine-phpfpm/start.sh

```diff
@@ -1,8 +1,19 @@
 #!/bin/sh
 
-cd /application
+IS_DEV=$(ping -c 1 -t 5 169.254.169.253 > /dev/null; echo $?)
```

Comment:
uhmmm .. it's indirect 🤔  I'd prefer an ENV variable "ENABLE_CW" instead .. you are this way inferring a behaviour basing your decision on a supposition

Reply:
fixed